sdcm/cluster_k8s/mini_k8s.py

```diff
@@ -653,3 +654,36 @@ def upgrade_scylla_cluster(self, new_version: str) -> None:
     @staticmethod
     def fstrim_scylla_disks():
         LOGGER.warning("Local K8S backends don't support running 'fstrim' command. Ignoring.")
+
+    def prefill_cluster(self, dataset_size: str):
+        if dataset_size and dataset_size not in FUNCTIONAL_TEST_DATASETS:
+            raise ValueError(f"Dataset size unexpected value: '{dataset_size}'. "
+                             f"Dataset size expected values are: 'small', 'medium', 'big'")
+
+        test_keyspace_name = 'keyspace1'
+        test_table_name = 'standard1'
+
+        replication_factor = len(self.nodes)
+        node = self.nodes[0]
+        test_keyspaces = self.get_test_keyspaces()
+        with self.cql_connection_patient(node=node) as session:
+            # if keyspace exists, drop it and create again
+            if test_keyspace_name in test_keyspaces:
+                session.execute(f"DROP KEYSPACE {test_keyspace_name}")
+
+            session.execute("""
+                CREATE KEYSPACE %s WITH replication = {'class': 'SimpleStrategy',
+                'replication_factor': %s}
+                """ % (test_keyspace_name, replication_factor))
+            session.execute(f"""
+                CREATE TABLE {test_keyspace_name}.{test_table_name}
+                (key blob PRIMARY KEY,"C0" blob,"C1" blob,"C2" blob,"C3" blob,"C4" blob)
+             """)
```

Comment:
it's not directly to this PR directly, but it's weird those CQL statements are not coming with the dataset. (the schema.cql is something we have for a long time now, with each snapshot)

Reply:
fixed